bin/run_fs_recon.py

```diff
@@ -122,10 +121,11 @@ def _run(subjects_dir, subject, raw_dir, force, mp, volume):
     run_subprocess(['mri_concat', '--rms', '--i', mri,
                     '--o', op.join(subjects_dir, subject, 'mri/orig/001.mgz')],
                    env=this_env)
-    run_subprocess(['recon-all', '-openmp', mp, '-subject', subject, '-all'], env=this_env)
+    run_subprocess(['recon-all', '-openmp', '%.0f' % mp, '-subject', subject, '-all'], env=this_env)
```

Comment:
I have a feeling you tried passing `mp` itself, not cast to a `str` already. `run_subprocess` wants them all to be strings. You might have also tried `%f` or something that would have made it a float and thus given the string `'1.0'` or something, which would fail at the `recon-all` stage